### PR TITLE
[v1] Update interpolate `outputRange` type to allow strings

### PR DESCRIPTION
**Pull request by chj-damon**
## Checklist

- [ ] Included code example that can be used to test this change
- [x] Updated TS types
- [x] Added TS types tests
- [ ] Added unit / integration tests
- [ ] Updated documentation
- [x] Ensured that CI passes

### CODE REVIEW

File: react-native-reanimated-tests.tsx [hunk 1]
```diff
@@ -464,3 +465,11 @@ function interpolateColorTest() {
 
   return null;
 }
+
+function interpolateNodeTest() {
+  const value = useValue(0);
+  interpolateNode(value, {
+    inputRange: [0, 1],
+    outputRange: ['0%', '100%'],
```
Comment (jakub-gonet) [inline react-native-reanimated-tests.tsx:473]: We only support interpolating degrees in the string form.

```suggestion
    outputRange: ['0deg', '360deg'],
```

File: react-native-reanimated.d.ts [hunk 2]
```diff
@@ -420,7 +420,10 @@ declare module 'react-native-reanimated' {
     export function cancelAnimation<T extends SharedValue<SharedValueType>>(
       sharedValue: T
     ): void;
-    export function withDelay(delayMS: number, delayedAnimation: number): number;
+    export function withDelay(
+      delayMS: number,
+      delayedAnimation: number
+    ): number;
```
Comment (jakub-gonet) [inline react-native-reanimated.d.ts:426]: Unrelated change.

**Review by jakub-gonet — CHANGES_REQUESTED** [review]
LGTM, left some minor comments.

**Comment by chj-damon** [comment]
updated.

**Review by jakub-gonet — APPROVED** [review]
Looks great, thanks!

**Pull request merged**